requirements.txt

```diff
@@ -5,3 +5,4 @@ django-apptemplates>=1.4,<2.0
 django-celery-email>=3.0.0,<3.1
 djangorestframework>=3.11,<3.12
 swapper
+cryptography<2.9.0,>=2.4.0
```

Comment:
not sure why this is needed, it should be already specified somewhere upstream. Can you try commenting it out and let the build run without it please?